keyboards/planck/keymaps/pok3r/Makefile

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
+	include ../../../../Makefile
+endif
```

Comment:
This file isn't used or needed anymore - would you mind deleting it?

Reply:
You're right, I forgot to remove it. Thanks!